docs/keystone-auth.md

```diff
@@ -0,0 +1,38 @@
+An authentication backend for Openstack's Keystone is maintained in this
+source tree as `haas.ext.auth.keystone`. This document describes its
+usage.
+
```

Comment:
Stating we use the Keystone v3 api should be sufficient.  It's up to the Keystone people to make sure they actually maintain that API.